bootloader/grub.go

```diff
@@ -483,3 +483,59 @@ func (g *grub) TrustedAssets() ([]string, error) {
 		"EFI/boot/grubx64.efi",
 	}, nil
 }
+
+// RecoveryBootChain returns the chain of recovery mode boot files for the
+// specified recovery system and kernel. This function should be called
+// only for the recovery bootloader.
```

Comment:
we don't need the comment I think, or it should be a copy of what we put in the interface.